_includes/404.html

```diff
@@ -0,0 +1,59 @@
+<section id="404">
+    <div class="container">
+        <div class="row">
+            <div class="col-lg-12 text-center">
+                <h2 class="section-heading">Error 404 - Page not Found!</h2>
+                <hr class="primary">
+                <p>Oh, snap! Clearly, this is not the page you're looking for. The page may have been renamed or removed.</p>
```

Comment:
Thanks. :) I tried to make the tone of the writing similar to the rest of the site.